sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/KeyOperationParameters.java

```diff
@@ -24,6 +24,24 @@
     @JsonProperty(value = "value", required = true)
     private Base64Url value;
 
+    /**
+     * Initialization vector for symmetric algorithms.
+     */
+    @JsonProperty(value = "iv")
+    private byte[] initializationVector;
```

Comment:
In public APIs, refer to this only as "iv". It's a common enough acronym in cryptography.